examples/nat/main/nat.go

```diff
@@ -28,6 +28,7 @@ func main() {
 	configFile := flag.String("config", "config.json", "Specify config file name")
 	flag.BoolVar(&nat.CalculateChecksum, "csum", true, "Specify whether to calculate checksums in modified packets")
 	flag.BoolVar(&nat.HWTXChecksum, "hwcsum", true, "Specify whether to use hardware offloading for checksums calculation (requires -csum)")
+	dpdkLogLevel := flag.String("dpdk", "--log-level=0", "Passes an arbitrary argument to dpdk EAL")
```

Comment:
Better to write "specific" instead of "arbitrary", because that is what you do, you pass a specific argument to EAL.

Reply:
It is possible to pass a single arbitrary argument to DPDK EAL. The default value "--log-level=0" is specific, but it is possible to change it to anything else, e.g. not related to log level.

Reply:
Yes, but that is what specific means. Not arbitrary.

Reply:
Help message explains what this command line switch does. It allows user to pass an arbitrary switch string DPDK:

  -dpdk string
        Passes an arbitrary argument to dpdk EAL (default "--log-level=0")